### PR TITLE
Implement wizard setup

### PR DESCRIPTION
**Pull request by joecohens**
@cachethq/owners Let me know what you think :)

https://github.com/cachethq/Cachet/issues/101

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Looks great to me @joecohens! This will be great for when we want to start adding more default settings.

**Comment by joecohens** [comment]
:)

**Comment by jbrooksuk** [comment]
Perhaps we allow them to update a banner during the setup?

**Comment by joecohens** [comment]
@jbrooksuk I can implement it if you like :) not sure if it's the right moment to doit since it's not required to setup.

File: app/lang/en/setup.php [hunk 1]
```diff
@@ -7,5 +7,7 @@
     'status_page_setup' => 'Status Page Setup',
     'show_support'      => 'Show support for Cachet? <small>Places a link in the footer linking to Cachet.</small>',
     'admin_account'     => 'Administrator Account',
-    'complete_setup'    => 'Complete Setup'
+    'complete_setup'    => 'Complete Setup',
+    'completed'         => 'Cachet has been configured successfully!',
```
Comment (GrahamCampbell) [inline app/lang/en/setup.php]: needs to be added in the fr file too

Comment (GrahamCampbell) [inline app/lang/en/setup.php]: just add the english translation. we do this to keep the files perfectly synced, formatting and all

Comment (joecohens) [inline app/lang/en/setup.php]: Ok sure!

File: app/lang/fr/setup.php [hunk 2]
```diff
@@ -8,4 +8,6 @@
     'show_support'      => 'Montrer votre support pour Cachet ?',
     'admin_account'     => 'Compte administrateur',
     'complete_setup'    => 'Terminer l\'installation'
```
Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: missing comma

Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: that's why we use trailing commas on multiline arrays, lol

File: app/routes/dashboard.php [hunk 3]
```diff
@@ -8,7 +8,7 @@
     ]);
 
     // Components
-    Route::group(['prefix' => 'components'], function() {
+    Route::group(['prefix' => 'components'], function () {
```
Comment (GrahamCampbell) [inline app/routes/dashboard.php]: this needs rebasing against the master

Comment (GrahamCampbell) [inline app/routes/dashboard.php]: this is already fixed there

File: app/lang/fr/setup.php (hunk 2 above)
Comment (joecohens) [inline app/lang/fr/setup.php]: Lol thanks for that!

**Comment by joecohens** [comment]
I think this si done if it's ok @jbrooksuk merge at will :)

**Comment by GrahamCampbell** [comment]
Probably should be squashed. Otherwise, :+1: from me. :)

**Comment by jbrooksuk** [comment]
Yup. Go for it Joe!

**Comment by joecohens** [comment]
I'll squash and merge :)

**Pull request merged**